src/controllers/threadController.js

```diff
@@ -74,9 +76,42 @@ export const getTotalThreadNumber = async (req, res, next) => {
 export const updateThread = async (req, res, next) => {
   try {
     console.log('Server::Updating a thread - running updateThread');
-    const id = req.params.id;
-    const updated = await ThreadService.updateThread(id, req.body);
-    res.status(200).json({ message: 'Successfully updated', updated });
+    // console.log('req: ', req);
+    const threadId = req.params.id;
+    const formBody = req.body;
+    console.log('threadId: ', threadId);
+    console.log('formBody: ', formBody);
+
+    // Separate thread data and pet data
+    const threadData = {
+      title: formBody['thread-title'],
+      content: formBody['thread-main-content'],
+      kind: formBody['select-thread-type']
+    };
+
+    const petData = {
```

Comment:
i guess the better approach is to change the name of these properties in the frontend.

Reply:
@ddm1293 The reason I didn't change the FE naming is you're still working on the latest (and hopefully final) version of the createForm. I'd like to change it after we have a stable version of the form in case there would be any potential conflicts or cause any merging mess-up.